packages/ListBoxBrowser/src/ListBoxBrowser.js

```diff
@@ -0,0 +1,355 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { container, flex } from "./ListBoxBrowser.styles";
+import {
+  getData,
+  getOptionByKey,
+  onChange as onChangeHelper,
+  isRoot,
+  focusListBoxBrowser,
+  focusListBoxRoot,
+  getInitialView,
+  extractedExtendedProps,
+  getDataOptionByFn,
+} from "./helpers";
+import OptionsSelected from "./components/OptionsSelected";
+import CustomListBox from "./components/CustomListBox";
+import Root from "./components/Root";
+import Browser from "./components/Browser";
+import Title from "./components/Title";
+
+const propTypes = {
+  /**
+    An array of javascript objects holding the data structure for the ListBoxBrowser
+    The object shape **must have** at least a string **label** property and an optional array **options** property.
+    Also can hold any other kind of data for your own use.
+    ex.
+
+    [{ label: "One" }, { label: "Two", options: [{ label: "Three" }] }]
+  */
+  data: PropTypes.any.isRequired, // eslint-disable-line
```

Comment:
Should it be `any`? Is there a way we can use prop-types to enforce (must have a label).

Reply:
yeap, we have to use a custom prop, don't know if it's that relevant to have it, I'm throwing an error already if I don't find the label. what do you think?